db/migrate/20221018144521_add_legacy_flag_to_invoices.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class AddLegacyFlagToInvoices < ActiveRecord::Migration[7.0]
+  def up
+    change_table :invoices do |t|
```

Comment:
Rails/BulkChangeTable: You can combine alter queries using `bulk: true` options.